nservicebus/upgrades/acs-host-6to7.md

```diff
@@ -8,6 +8,42 @@ related:
 - nservicebus/upgrades/5to6
 ---
 
+## Namespace changes
+
+All types specified in this document are now in the `NServiceBus` namespace.
+
+## IConfigureThisEndpoint changes
+
+`IConfigureThisEndpoint.Customize` is passed an instance of `EndpointConfiguration` instead of `BusConfiguration`.
+
+snippet: AzureServiceBusTransportWithAzureHost
+
+## AsA_Host role changes into IConfigureThisHost
+
+The specifiers `IConfigureThisEndpoint` and `AsA_Host` are now merged into `IConfigureThisHost`. 
```

Comment:
The model was really botched up in the past, you had to specify IConfigureThisEndpoint, but leave the implemenation of it blank as any changes to it had no effect whatsoever. (note this method was added far later than the role model was). At the same time you also had to specify the role.